proto/kvrpcpb.proto

```diff
@@ -817,39 +821,76 @@ enum IsolationLevel {
     RC = 1; // RC = read committed
 }
 
-message HandleTime {
-    // Time spent in queue.
-    int64 wait_ms = 1;
-    // Processing time, excluding wait time.
-    int64 process_ms = 2;
+message TimeDetail {
+    // Off-cpu wall time elapsed in TiKV side. Usually this includes queue waiting time and
+    // other kind of waitings in series.
+    int64 wait_wall_time_ms = 1;
+    // Off-cpu and on-cpu wall time elapsed to actually process the request payload. It does not
+    // include `wait_wall_time`.
+    // This field is very close to the CPU time in most cases. Some wait time spend in RocksDB
```

Comment:
Do we really have the ability to measure the CPU time of a single request, even we don't take Mutex wait time into account?

Reply:
The precise CPU time can be measured by user_cpu_time + system_cpu_time. Not sure about the cost though.

Reply:
IMO, this field is not close to CPU time. It's a typical wall time.

Reply:
Oh yes. I only want to express that, in 99% cases, the higher this field is, the higher CPU usage there will be.